docs/install/provision.rst

```diff
@@ -50,29 +50,29 @@ At this point you have the master server set-up and preloaded with content. Befo
 	``<username>`` refers to the user you were logged in as when you installed Kolibri as explained above. In the image below for example, you can see that the user who installed Kolibri is called **IEUser**.
 
 	.. figure:: /img/IEUser.gif
-	    :alt:  
+	    :alt:
```

Comment:
Actually, the empty space is here for the reason, it translates to an empty `alt` attribute for a given image, allowing the screenreaders to ignore it. It's used in several images in the documentation where the wording describing the image in the `alt` would be repeating a lot of information that is already present in the text above or below the image.

![Provisioning many servers — Kolibri 0 13 documentation - Google Chrome_437](https://user-images.githubusercontent.com/1457929/80042373-1742c480-84ff-11ea-8991-f3df3a530085.png)

Reply:
Hm, that's unfortunate. Trailing whitespace is safe to trim in most other markup and coding languages, and most developers have a setting that automatically clears it.

Could this be achieved inserting a non-breaking space? Something like:

```rst
	.. figure:: /img/IEUser.gif
	    :alt: |nbsp|
```

Reply:
I remember checking various options some time ago, and only thing that worked was an actual whitespace value there, but apparently something changed, and now a proper empty `alt` attribute is produced with and without it.

![LEDev20 (start)  Running  - Oracle VM VirtualBox_443](https://user-images.githubusercontent.com/1457929/80163889-d10d6400-85d7-11ea-9e48-71de9014fe3b.png)

Thank you for making me test this, it will make things easier!